python/server/ServerWrapper.cpp

```diff
@@ -321,6 +321,20 @@ namespace {
         return py_items;
     }
 
+    // Wrapper for starting buildings
+    list LoadStartingBuildings() {
```

Comment:
If you call this function for every empire, isn't it going to re-load and parse them all for every empire still? Thought part of the point was to avoid doing the disk access repeatedly.